test/e2e/template/helmfile/testdata/snapshot/release_template_inheritance/input.yaml

```diff
@@ -12,7 +12,7 @@ templates:
     values:
     - template1: template1
     valuesTemplate:
-    - template1Label: "{{` '{{ .Release.Labels.template1 }}' `}}"
```

Comment:
I'm not completely sure, but goccy/go-yaml seemed to translate single quotes enclosed in double quotes to triple single quotes, which resulted in this test case failing.
You don't need the single quotes anyway to write a valid helmfile yaml for this configuration so I just omitted it.